packages/syntax-core/src/RadioButton/RadioButton.tsx

```diff
@@ -71,7 +71,7 @@ const RadioButton = ({
   /**
    * Value of the selected radio option
    */
-  value: string;
+  value?: string | readonly string[] | number | undefined;
```

Comment:
@JFChoe `string` or `number` make sense but could you explain when we need `string[]` or `undefined`?

Reply:
Sorry, I just added it in cuz I saw the primitive can take those data types too. 

Let me take it out tho